app/scripts/components/topic.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { PropTypes } from 'react';
+
+import ScoreBar from './score_bar';
+import X from './icons/x';
+
+const Topic = ({ id, topic, red, amber, green, remove }) => {
+  const colors = [{ red }, { amber }, { green }];
```

Comment:
You can't map over an object to render 😢  I'm not too keen on this bit but struggled to find a nice solution.